src/config.ts

```diff
@@ -23,23 +24,11 @@ export interface Flags {
 export async function getConfig(flags: Flags) {
   // check to see if a config file path was passed
   const configPath = flags.config || 'linkinator.config.json';
-  let configData: string | undefined;
-  try {
-    configData = await fs.readFile(configPath, {encoding: 'utf8'});
-  } catch (e) {
-    if (flags.config) {
-      console.error(`Unable to find config file ${flags.config}`);
-      throw e;
-    }
-  }
-
   let config: Flags = {};
-  if (configData) {
-    config = JSON.parse(configData);
+
+  if (flags.config) {
+    config = await parseConfigFile(configPath);
   }
-  // `meow` is set up to pass boolean flags as `undefined` if not passed.
```

Comment:
This comment is probably worth keeping around for future me please :)

Reply:
The comment is restored @JustinBeckwith